assets/js/components/HostDetails/HostDetails.test.jsx

```diff
@@ -66,4 +67,115 @@ describe('HostDetails component', () => {
       )
     ).toBeInTheDocument();
   });
+
+  it("should not display 'Clean up' button when host is not deregisterable", () => {
```

Comment:
We could maybe add a `deregistration` describe to group the tests about this topic.
Again, not strong opinions

Reply:
+1